src/main/java/calculator/operator/Addition.java

```diff
@@ -0,0 +1,8 @@
+package calculator.operator;
+
+public class Addition implements Operator {
+    @Override
+    public Long action(Long x, Long y) {
+        return x+y;
```

Comment:
```suggestion
        return x + y;
```
연산자 사이에 공백이 있으면 가독성이 좀 더 확보될 거 같아요~!